### DISCUSSION TITLE
Loss = nan why?

### DISCUSSION THREAD

**Discussion by zhangqianYY** [hpcaitech/ColossalAI#1114] [Community | Q&A] [answered]
hi, I run (image/vision_transformer/data_parallel/train_with_cifar10.py) `colossalai run --nproc_per_node 1  train_with_cifar10.py  --config config.py`    ,when BATCH_SIZE = 2048,gradient_accumulation=240 ,  lead to Loss = nan,How can it optimize?

**Comment by BoxiangW** [comment, marked as answer]
Hi, you could refer to [discussion #1116](https://github.com/hpcaitech/ColossalAI/discussions/1116).
